deflect/types.h

```diff
@@ -71,6 +71,16 @@ enum class RowOrder
     bottom_up /**< OpenGL image with (0,0) at the bottom-left corner. */
 };
 
+/** The possible formats for image data. */
+enum class Format : std::uint8_t
+{
+    rgba = 0, // equivalent to old compressed=false property
```

Comment:
with 1.0, maybe those references to old things are no longer needed to be mentioned

Reply:
You're right, it probably doesn't serve a purpose anymore. Anyone changing this enum will break the network protocol, but that's true of all the other enums as well, so no reason to mark those two as "special".